strapi/strapi
# Transfer assets to and from external providers

### What does it do?

- Use the upload provider to upload assets
- Transfer assets to or from external providers

### Why is it needed?
- to be able to export, import or Transfer files from or to the different providers

### How to test it?
- try the transfer features (pull, push, import and export) with different upload providers (local provider, aws provider and cloudinary provider)
- you should be able too export assets from different providers
- you should be able to transfer your assets to your providers successfully

fixes #15940
